lisp/Makefile

```diff
@@ -53,8 +54,18 @@ uninstall:
 	then \
 		cd $(SITELISP) && $(UNINSTALL) ess-site.el ; \
 	fi;
+
+ess-autoloads.el: $(ELC)
+	@printf	"\nGenerating $@\n"
+	$(EMACSBATCH) --eval "(progn\
+	(setq make-backup-files nil)\
+	(setq generated-autoload-file (expand-file-name \"$@\"))\
+	(setq find-file-visit-truename t)\
+	(update-directory-autoloads default-directory))"
+
 clean:
 	rm -f $(ELC)
+	rm -f ess-autoloads.el
```

Comment:
I am working from source and this cleanning of .el files interferes with this pattern. I just removed the cleaning of julia-mode.el for this reason. I wonder if we should not clean ess-autoloads.el here either. 

I am actually thinking about adding an extra target. Either cleanall, or cleanelc. Probably the latter. WDYT?

Reply:
Fine with me. `cleanelc` is probably clearest. If you add that I can rebase this PR.